md/message-states.md

```diff
@@ -1,6 +1,27 @@
-# Message states
+# SMS message states
 
-All messages in the system are in one of the following states.
+## Interaction between medic-webapp and medic-gateway
+[Medic-webapp](https://github.com/medic/medic-webapp) uses [medic-gateway](https://github.com/medic/medic-gateway)
+to send SMS messages.
+
+When a report comes in, [medic-sentinel](https://github.com/medic/medic-sentinel) adds the appropriate list of
+scheduled messages (to be sent at a future date) to the report doc.
+
+Periodically, sentinel checks for messages that need to be sent, and [sets their status to `pending`](https://github.com/medic/medic-sentinel/blob/master/schedule/due_tasks.js).
+
+Periodically, the gateway pings the webapp over HTTP to get any messages that need to be sent.
+and report on the status of the messages it's trying to send.
```

Comment:
> the messages it's trying to send

...and the messages that it has already sent, and for which now updates on the delivery status are available.